test/e2e/logstash/logstash_test.go

```diff
@@ -87,6 +90,9 @@ func TestLogstashWithCustomServiceAndAmendedApi(t *testing.T) {
 
 	logstashBuilder := (logstash.NewBuilder(name).
 		WithNodeCount(1).
+		WithConfig(map[string]interface{}{
```

Comment:
nit: Could we add a comment/godoc on `TestLogstashWithReworkedApiService` and `TestLogstashWithCustomServiceAndAmendedApi`? It took me a few minutes to figure out why this change is not needed for `TestLogstashWithCustomService`.